roles/hosted_engine_setup/tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -53,6 +53,13 @@
     with_items:
       - "OVESETUP_DWH_DB/password=str:{{ he_dwh_db_password }}"
     when: he_dwh_db_password is defined
+  - name: Add keycloak line to answerfile
+    lineinfile:
+      path: /root/ovirt-engine-answers
+      line: "{{ item }}"
+    with_items:
+      - "OVESETUP_KEYCLOAK_CORE/enable=bool:{{ he_enable_keycloak }}"
+    when: he_enable_keycloak is defined
```

Comment:
`he_enable_keycloak` is always defined since you added it to `defaults/main.yml`

Reply:
Removed